src/components/courseTimeline.tsx

```diff
@@ -0,0 +1,57 @@
+const CourseTimeline = () => {
+  return (
+    <div className="mx-auto flex w-[140vh] flex-col items-center justify-start gap-[2vh] rounded-xl bg-[#023851] px-[12vh] py-[6vh] font-malayalam text-[#FFF4B8]">
+      <h1 className="text-center text-[3.25vh]">Timeline of Course</h1>
+
+      <div>
+        <h2 className="mb-[2vh] text-[3.25vh]">Lorem Ipsum</h2>
+        <ul className="relative ml-[2vh] space-y-[4vh] pl-[4vh] *:text-[1.75vh] *:before:absolute *:before:left-0">
```

Comment:
```suggestion
        <ul className="relative ml-[1.5vw] space-y-[4vh] pl-[2vw] text-[1.75vh] *:before:absolute *:before:left-0">
```
vh is for viewport height, you're moving it to the left (x axis) you should be using vw